volumes/ecocounter/tables/counts.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE ecocounter.counts (
```

Comment:
We would ideally partition this in years but it is good for meow

Reply:
Only a couple million records at the moment though. Is there a threshold where partitioning starts to make sense?